kotsadm/web/src/components/apps/AppDetailPage.jsx

```diff
@@ -422,6 +424,14 @@ class AppDetailPage extends Component {
                     <Route exact path="/app/:slug/snapshots/:id/restore" render={() =>
                       <AppSnapshotRestore app={app} />
                     } />
+                    {isIdentityServiceSupported &&
+                      <Route exact path="/app/:slug/settings" render={() =>
+                        <AppSettings
```

Comment:
don't wanna be picky, but why not call it `AppIdentityServiceSettings` for example?

Reply:
Yeah, will update the component name